src/commands/deploy.js

```diff
@@ -1,131 +1,18 @@
-const chalk = require('chalk');
```

Comment:
the deleted code in this module has been split between the apiMiddleware and the new standalone `create` command